file_processing/database.py

```diff
@@ -14,14 +14,46 @@ def insert_into_db(data):
     conn = connect_to_db()
     cursor = conn.cursor()
     
+    sql = "INSERT INTO receipts (receipt_name, date) VALUES (%s, %s);"
```

Comment:
this method "insert_into_db" also have generic name, please rename following the same approach as for other methods in this class